Document/0x06b-Basic-Security-Testing.md

```diff
@@ -359,7 +372,47 @@ Possible values for the property [UIDeviceFamily](https://developer.apple.com/li
 
 The following sections describes on how to retrieve basic information of an iOS app, that might be useful during a penetration test.
 
--- ToDo Objection: https://github.com/OWASP/owasp-mstg/issues/1247
+##### Mobile Security Framework (MobSF)
+
+MobSF is a penetration testing framework that is capable of analysing IPA files and can be used before even installing the app on your testing device.
+
+Once you have MobSF up and running you can open it in your browser by navigating to <http://127.0.0.1:8000>. Just select the IPA you want to analyse and MobSF will start it's job. The bigger the app the longer it takes, but usually you should get some feedback within a few minutes.
+
+The result screen might first be overwhelming but will have a very good overview of the potential attack surface and has automated a few things for you. With MobSF you can now for example:
```

Comment:
Little bit weird. How about:

After MobSF is done with its analysis, you will receive a one-page overview of all the tests that were executed. While it may look daunting at first, the page is split up into multiple sections, each with their own purpose. Together, all the sections give a good first indication of the attack surface of the application. You can also execute additional actions, such as:

(Sorry, let myself go :) )

Reply:
reads much better :-) It's in